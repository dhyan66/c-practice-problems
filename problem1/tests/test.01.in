new 30
set 1
set 3
reset 3
set 5
set 10
reset 5
set 11
end
