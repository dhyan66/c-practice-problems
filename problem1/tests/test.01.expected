00 00 00 00 
02 00 00 00 
0a 00 00 00 
02 00 00 00 
22 00 00 00 
22 04 00 00 
02 04 00 00 
02 0c 00 00 
